test/interop/fortran/fortArrayToDRAutomatic/chapelProcs.chpl

```diff
@@ -0,0 +1,12 @@
+export proc chpl_library_init_ftn() {
+  extern proc chpl_library_init(argc: c_int, argv: c_ptr(c_ptr(c_char)));
+  var filename = c"fake";
+  chpl_library_init(1, c_ptrTo(filename): c_ptr(c_ptr(c_char)));;
+  chpl__init_chapelProcs();
+}
```

Comment:
It might be good to generate this for the user eventually (separate task for later)